tensorflow/lite/micro/examples/hello_world/train/train_hello_world_model.py

```diff
@@ -468,8 +468,6 @@ def representative_dataset():
   converter.optimizations = [tf.lite.Optimize.DEFAULT]
   # Enforce integer only quantization
   converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS_INT8]
-  converter.inference_input_type = tf.int8
-  converter.inference_output_type = tf.int8
```

Comment:
Is the checked in hello_world.tflite the new output of conversion using this script?

Reply:
Yes, the hello_world.tflite file that is part of the PR is the output of the this conversion.